gtars/src/uniwig/mod.rs

```diff
@@ -430,10 +552,98 @@ pub fn uniwig_main(
                 }
             }
         }
+        }
+    );
+
+    let vec_strings = vec!["start", "core", "end"];
+
+    match output_type {
+        "wig" => {
+            println!("Combining Wig Files");
+
+            for location in vec_strings.iter(){
+
+                write_combined_wig_files(*location, output_type, bwfileheader, &chromosomes);
+
+            }
+
+
+
+        }
+        _ => {
+
+        }
+
     }
+
+
+    println!("FINISHED");
+
     Ok(())
 }
 
+fn fixed_core_wiggle_bam(p0: &Vec<i32>, p1: &Vec<i32>, p2: i32, p3: i32) -> (Vec<u32>, Vec<i32>) {
```

Comment:
All of these variables are passed in, but do they do anything?

Also I am pretty sure you can just use slices here... otherwise you might be creating a whole new struct for just passing by reference...

So maybe this instead:
```rust
fn fixed_core_wiggle_bam(p0: &[i32], p1: &[i32], p2: i32, p3: i32) -> (Vec<u32>, Vec<i32>)
```

Reply:
these two functions are place holders, i'm still working on bam functionality